sbt-twirl/src/main/scala/play/twirl/sbt/SbtTwirl.scala

```diff
@@ -74,11 +73,16 @@ object SbtTwirl extends AutoPlugin {
   )
 
   def dependencySettings = Def.settings(
-    // Task is undefined if not a ScalaJSPlugin
-    isScalaJSProject := (isScalaJSProject ?? false).value,
     twirlVersion := readResourceProperty("twirl.version.properties", "twirl.api.version"),
-    // %%% will be the same as %% for normal projects
-    libraryDependencies += "com.typesafe.play" %%% "twirl-api" % twirlVersion.value
+    libraryDependencies += {
+      val crossVer = crossVersion.value
+      val isScalaJS = CrossVersion(crossVer, scalaVersion.value, scalaBinaryVersion.value) match {
+        case Some(f) => f("").contains("_sjs0.6") // detect ScalaJS CrossVersion
+        case None => false
+      }
+      val baseModuleID = "com.typesafe.play" %% "twirl-api" % twirlVersion.value
```

Comment:
I didn't checked it, but besides adding the build target we are probably ready for scala-js 1.0